src/Components/TrainingListPage.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, Image, TouchableOpacity, FlatList } from 'react-native';
+import { connect } from 'react-redux';
+import theme from '../styles/theme.style';
+
+const getTrainingDuration = trainingDrills => {
+  const durationList = trainingDrills.map(({ durationInMinutes }) => durationInMinutes);
+  const totalDurationInMinutes = durationList.reduce((a, b) => a + b, 0);
+  return totalDurationInMinutes;
+};
+
+const getTrainingNbPlayers = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
+  const totalNbPlayers = Math.max(...nbPlayersList);
+  return totalNbPlayers;
+};
+
+const mapStateToProps = state => {
+  return {
+    trainings: state.trainings,
+    allDrills: state.drills,
+  };
+};
+
+export const TrainingListPage = props => {
+  const { allDrills, trainings, navigation } = props;
+
+  const [data] = useState(trainings);
+
+  return (
+    <View style={styles.trainingListPage}>
+      <Text style={styles.counter}>{data.length} training sessions available</Text>
+      <FlatList
+        data={data}
+        keyExtractor={item => item.id.toString()}
+        renderItem={({ item }) => (
+          <TouchableOpacity
+            style={styles.training}
+            onPress={() => navigation.navigate('TrainingPage', { training: item })}
+          >
+            <Image style={styles.image} source={{ uri: item.image }} />
```

Comment:
If you have a look at #54 you could implement the same modification on image opacity

Reply:
Sounds good. What do you think is the relevant information for training sessions ? Duration or nb player ?

Reply:
I have no idea 🤷‍♂ 
Maybe we should let it empty here? The simpler the better

Reply:
So I tried, but without anything written on the image the result is kind of ugly.
Can I show you tomorrow so we can decide what to do ?

Reply:
In the end I put both information. I like the idea of using the image space.
I guess we can still change later if we think only part of the information (or none) is relevant.